### ISSUE TITLE
Jemm386 v5.79 crashes when UMBPCI is used

### ISSUE THREAD

**Issue by Baron-von-Riedesel** [Baron-von-Riedesel/Jemm#2]
If the umbpci.sys driver is loaded and supplied UMBs for DOS, loading Jemm386 most likely will cause a crash, since v5.79 does no longer scan for RAM in the upper region if the BIOS supports int 15h, ax=e820h (ACPI).

workaround: use the X=xxxx-xxxx to tell Jemm386 what region to exclude

**Comment by Baron-von-Riedesel** [comment]
One may also change the source INIT16.ASM:

```
if 0;?INTEGRATED    ;<---- change 0 to 1
;--- don't scan for RAM if we called Int 15h ax=e820h
	cmp bI15RsvdChk,0
	jnz @F
endif
```

**Comment by Baron-von-Riedesel** [comment]
it's fixed in v5.80
